Sources/ForageSDK/Foundation/Telemetry/ForageLogger.swift

```diff
@@ -116,6 +123,10 @@ internal class DatadogLogger : ForageLogger {
             in: datadogInstance
         )
         
+        // Do this explicitly in the constructor so that we confirm each logger that is created
+        // has the trace_id field.
+        self.logger?.addAttribute(forKey: "trace_id", value: DatadogLogger.traceId)
```

Comment:
should we add a line like this to ensure the Trace ID is visible in the attributes/context section of a log?

```swift
        self.logger?.addAttribute(forKey: "forage_trace_id", value: DatadogLogger.traceId)
```

Reply:
I think it's fine without it, it's appended to the each log statement already. It just appears under the trace tab rather than as an attribute.